frb_codegen/src/generator/dart/ty_delegate.rs

```diff
@@ -60,6 +73,27 @@ impl TypeDartGeneratorTrait for TypeDelegateGenerator<'_> {
             IrTypeDelegate::PrimitiveEnum { ir, .. } => {
                 format!("return {}.values[raw];", ir.dart_api_type())
             }
+            #[cfg(feature = "chrono")]
+            IrTypeDelegate::Time(ir) => match ir {
+                IrTypeTime::Local => "
+                if (identical(0, 0.0)) {
```

Comment:
Since this mostly doesn't change, we can extract this as a constant.

Reply:
Or maybe put it in `frb_dart` support lib

Reply:
resolved in [9b020a2](https://github.com/fzyzcjy/flutter_rust_bridge/pull/694/commits/9b020a25756f2c45c99f0b88265070274e6e934f)